src/cascadia/TerminalSettingsEditor/AddProfile.xaml

```diff
@@ -23,19 +23,15 @@
 
     <ScrollViewer ViewChanging="ViewChanging">
         <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <Button x:Uid="AddProfile_AddNewButton"
-                    AutomationProperties.AutomationId="AddProfile_AddNewButton"
-                    AutomationProperties.Name="{Binding Tag, RelativeSource={RelativeSource Self}}"
+            <Button x:Name="AddNewButton"
                     Click="AddNewClick"
                     Style="{StaticResource AccentButtonStyle}">
-                <Button.Content>

```

Comment:
sorry to be a pill, but can you not do this part if it is not part of the bug fix? It's harder to service bigger diffs... and you're going to conflict with #12326 as it is and if we can make the actual bug fix part more obvious we should